sqlmesh/core/renderer.py

```diff
@@ -202,19 +202,54 @@ def _resolve_tables(
         if not snapshots and not table_mapping and not expand:
             return expression
 
+        from sqlmesh.core.snapshot import to_table_mapping
+
         expression = expression.copy()
         with _normalize_and_quote(expression, self._dialect) as expression:
-            return _resolve_tables(
-                expression,
-                snapshots=snapshots,
-                table_mapping=table_mapping,
-                expand=expand,
-                deployability_index=deployability_index,
-                start=start,
-                end=end,
-                execution_time=execution_time,
-                **kwargs,
-            )
+            snapshots = snapshots or {}
```

Comment:
why is this no longer a function?

Reply:
it's not needed, it's not called anywhere outside of this instance method, and i needed  dialect